apps/omg_child_chain/lib/omg_child_chain/datadog_event/contract_event_consumer.ex

```diff
@@ -58,12 +58,12 @@ defmodule OMG.ChildChain.DatadogEvent.ContractEventConsumer do
   """
   def init(args) do
     publisher = Keyword.fetch!(args, :publisher)
-    topic = Keyword.fetch!(args, :topic)
+    {:root_chain, event_name} = topic = Keyword.fetch!(args, :topic)
     release = Keyword.fetch!(args, :release)
     current_version = Keyword.fetch!(args, :current_version)
     :ok = OMG.Bus.subscribe(topic, link: true)
 
-    _ = Logger.info("Started #{inspect(__MODULE__)} for event #{inspect(topic)}")
+    _ = Logger.info("Started #{inspect(__MODULE__)} for event #{event_name}")
```

Comment:
I don't like the change - it removes the context. In the previous PR we extended events with `:root_chain | :child_chain` and here we just log event name - it could increase the confusion.

What's wrong with former `#{inspect(topic)}`

Reply:
This module is intended only for contract events, that's why it's always `:root_chain`. This log message suits better in this context. 
The main reason for this PR is that in one of the GenServers we didn't have inspect and it was failing when starting on dev.

Reply:
What's obvious now won't necessary be in the month. 
- Also we specifically added `:root_chain | :child_chain` to defend against event names clashing. 
- Also, it's common to try to find logic behind just by searching the logged message.

Let's log `... root_chain:#{event_name}`